maps_dashboards/server/saved_objects/capabilities_provider.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const capabilitiesProvider = () => ({
+  map: {
+    // TODO: investigate which capabilities we need to provide
+    // createNew: true,
+    // createShortUrl: true,
+    // delete: true,
+    show: true,
```

Comment:
i think it's fine to set this to be true by default for now, in the future, this can be used for permission control if needed by the map plugin.

Reply:
Thanks, will add more content to part.